packages/terra-clinical-item-view/src/ItemView.scss

```diff
@@ -94,6 +94,7 @@
       // prematurely squished and maximize the available space for all labels.
       max-width: 60%;
       padding-left: 5px;
+      text-align: end;
```

Comment:
If `iconAlignment="inline" ` is not used, will this correctly right align? Or are the updated to add `iconAlignment="inline" ` for visuals on the doc site?

Reply:
Yes, the additional prop would be encouraged to be used (if not, the icon won't flow inline and will hang out to the left). 
+1 on updating the docsite example to include it or add a new one to especially call it out. Updated tests but would be good to have it documented as instructions.